code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -0,0 +1,39 @@
+#define ASH_WALKER_SPAWN_THRESHOLD 2
```

Comment:
`lavaland_run_code.dm` is an ambiguous name and can imply several things. I split each unique ruin's specific objects into different files to be more organized, and ash walker eggs aren't in this file because I put them into `ghost_role_spawners.dm`.